### PR TITLE
Use cfg default_wrap value for grid logs

### PR DESCRIPTION
**Pull request by pierrejeambrun**
Related: https://github.com/apache/airflow/issues/24721

Thanks @bbovenzi for the ping. Here is a quick PR that also uses the `default_wrap` param value for the grid logs.

### CODE REVIEW

File: airflow/www/templates/airflow/grid.html [hunk 1]
```diff
@@ -42,6 +42,7 @@
     const stateColors = {{ state_color_mapping|tojson }};
     const autoRefreshInterval = {{ auto_refresh_interval }};
     const defaultDagRunDisplayNumber = {{ default_dag_run_display_number }};
+    const defaultWrap = {{ default_wrap|tojson }};
```
Comment (bbovenzi) [inline airflow/www/templates/airflow/grid.html]: Does `tojson` convert to boolean?

Comment (pierrejeambrun) [reply airflow/www/templates/airflow/grid.html]: Yes, I get an error without it, he is trying to access a variable named `True` and `False` respectively.

Comment (pierrejeambrun) [reply airflow/www/templates/airflow/grid.html]: image
image

Comment (pierrejeambrun) [reply airflow/www/templates/airflow/grid.html]: I could replace this with something more explicit: (but still not ideal, I find the string quotes confusing)
```
    const defaultWrap = {{ 'true' if default_wrap else 'false' }};
```

Comment (bbovenzi) [reply airflow/www/templates/airflow/grid.html]: Actually, we should put this in a `<meta>` tag vs directly accessing here. and then in the js file we can call ```const defaultWrap = getMetaValue('default_wrap') === 'True';```

**Review by Kinceo1 — APPROVED** [review]
💡

File: airflow/www/templates/airflow/grid.html (hunk 1 above)
Comment (pierrejeambrun) [reply airflow/www/templates/airflow/grid.html]: Done :)

**Review by bbovenzi — APPROVED** [review]
lgtm!

**Pull request merged**